### ISSUE TITLE
Suggestion - Highlight favorited camos on weapons screen

### ISSUE THREAD

**Issue by TylersApps** [carlssonemil/orion#93]
**TLDR: It'd be nice to be able to find the camos I want to prioritize on the Camouflages page, favorite them, and then see them highlighted on the Weapons screen so that I can reference the requirements for each alongside the relevant weapon.**

---

I think that it would be a significant improvement to the UI/UX of the website. 

What I am thinking of could be as little as a gold outline (to make the user associate it with the gold star of favorites) around the _individual skins_ in the list or grid view on the Weapons page.

Personally, I am trying to grind out some of the camos in the "Fun" and "Solid Colors" categories specifically. The way it is currently set up, I have to favorite both the camo on the Camouflages page _and_ the relevant weapon on the Weapons page and then alternate between tabs in the site. I like the descriptions in the Weapons list view of each skin's requirements but it's harder to find individual skins on that page which is why I need to navigate to the Camouflages page frequently.

**Comment by carlssonemil** [comment]
Sorry for taking so long to reply, been a bit busy with vacations and other stuff this summer 😅 

I get what you mean, but the requirement (including the specific weapon) is available on the camouflage page, so what exactly would this solve that can't already be visible on each page? 🤔

Adding a small star next to each camouflage on the weapons page is certainly doable and I'll look into it for sure!

**Comment by carlssonemil** [comment]
I've decided to put this on ice, sorry.
